jbehave-core/src/main/java/org/jbehave/core/parsers/RegexStoryParser.java

```diff
@@ -59,6 +62,9 @@ public Story parseStory(String storyAsText, String storyPath) {
         Narrative narrative = parseNarrativeFrom(storyAsText);
         GivenStories givenStories = parseGivenStories(storyAsText);
         Lifecycle lifecycle = parseLifecycle(storyAsText);
+        if (lifecycle == null) {
+            meta = Meta.createMeta("@skip", keywords).inheritFrom(meta);
```

Comment:
No. If ```parameters``` are not specified all will work as it worked before, ExamplesTable will be set to the story as it is.
But if  ```parameters``` exist, ExamplesTable will be cut to contain only specified variables.